provider/semgrep/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@openctx/provider-semgrep",
+  "version": "0.0.12",
```

Comment:
if this was never published, should this be 0.0.1 instead?

Reply:
lol yes, I copied it from the `hello-world` provider directory. amending.